src/powershell/enable-rdp.ps1

```diff
@@ -28,3 +28,36 @@ if ($(Get-NetFirewallRule -Name $name).Enabled -ne $true) {
     Write-Error "[X] Failed to verify firewall rule: $name" -ErrorAction Stop
 }
 Write-Output "[*] Firewall rule successfully verified: $name"
+
+# Allow Terminal Server shadowing through the Windows firewall
+$name = "Remote Desktop - Shadow (TCP-In)"
+Write-Output "[ ] Configuring firewall rule: $name"
+Set-NetFirewallRule -DisplayName $name -Enabled True -Profile Private
+if ($(Get-NetFirewallRule -DisplayName $name).Enabled -ne $true) {
+    Write-Error "[X] Failed to verify firewall rule: $name" -ErrorAction Stop
+}
+Write-Output "[*] Firewall rule successfully verified: $name"
+
+# Enable Terminal Server shadowing
+$name = "Shadow"
+$path = 'HKLM:\SYSTEM\CurrentControlSet\Control\Terminal Server\WinStations\RDP-Tcp'
+Write-Output "[ ] Configuring setting: $name"
+Set-ItemProperty -Path $path -Name $name -Value "1"
+if ($(Get-ItemProperty -Path $path -Name $name).Shadow -ne 1) {
+    Write-Error "[X] Failed to verify setting: $name" -ErrorAction Stop
+}
+Write-Output "[*] Setting successfully verified: $name"
+
+# Give the Administrators group full control of Terminal Server
+Get-CimInstance -Namespace root\CIMV2\TerminalServices -ClassName Win32_TSPermissionsSetting -Filter 'TerminalName ="RDP-Tcp"' |
+        Invoke-CimMethod -MethodName AddAccount -Arguments @{AccountName="BUILTIN\Administrators"; PermissionPreSet="2"}
+
+# Enable Shadow Remote Desktop
+$name = "Terminal Services"
+$path = 'HKLM:\SOFTWARE\Policies\Microsoft\Windows NT\Terminal Services'
+Write-Output "[ ] Configuring setting: $name"
+Set-ItemProperty -Path $path -Name $name -Value "2"
+if ($(Get-ItemProperty -Path $path -Name $name).$name -ne 2) {
```

Comment:
Is "2" a magic number here?  Can the comment above (`# Enable Shadow Remote Desktop`) be expanded to explain why we are setting the value to 2 here?

Reply:
@dav3r resolved with [f9d5280](https://github.com/cisagov/windows-server-packer/pull/22/commits/f9d52805472f732eb349be69f7302867afeb7c65)

Reply:
Looks like you have the wrong commit hash - it should be https://github.com/cisagov/windows-server-packer/pull/22/commits/5567c51169366fbc9056b83a8e18967076fe0552.

Also, can you please confirm that both options have the same text, even though one is option 1 and the other is option 2:
```
# "Allow connections only from computers running Remote Desktop with Network Level Authentication (recommended)"
```

Reply:
hey @dav3r I apologize, both were the wrong comment. My AI autocomplete plugin overrode what I originally wrote. I updated them to right comment. I also changed the option for shadowing from prompt user's consent to not requiring a user's consent when attempting to shadow an active session. I believe that's the preferred option unless you believe otherwise. Resolved with [03f81cc](https://github.com/cisagov/windows-server-packer/pull/22/commits/03f81cc0fee10f0471689190525e61f13a2157b6)

Reply:
This option change will need to be tested so I'll push a prerelease after checks pass

Reply:
Looks good.  You are correct that we want to allow shadowing without requiring the user to confirm it.

Reply:
I'll leave this conversation open until your re-testing is successful.  Once it is, mention something here and I'll resolve it.

Reply:
@dav3r just created the pre-release: https://github.com/cisagov/windows-server-packer/releases/tag/v0.2.0-rc.3

Please run post terraform so I can access the AMI for testing. Thanks!